scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -569,7 +569,15 @@ public void copyScratchDataToOutput(List<OutputSelector> outputFiles) throws Fil
     }
 
     @Override
-    public void cleanScratchSpace() {
+    public void close() {
+        if (!executorTransfer.shutdownNow().isEmpty()) {
+            logger.error("Remaining tasks to execute while closing thread pool used for data transfer");
```

Comment:
It should never happen with the current code. There is a synchronization barrier in `TaskProActiveDataspaces` that forces to wait for all tasks submitted in the thread pool. The log is there to notice there is a problem if in the future the code is changed (e.g. the synchronization is changed or removed).